Source/rclUE/Public/ROS2Node.h

```diff
@@ -45,8 +51,8 @@ struct RCLUE_API FSubscription
 
     rcl_subscription_t rcl_subscription;
 
-    UPROPERTY(EditAnywhere, BlueprintReadWrite)
     FSubscriptionCallback Callback;
+    FSubscriptionCallbackBP CallbackBP;

```

Comment:
I think trying to have a separate delegate for Blueprint and C++ is going to cause confusion. I think it would be better to stick with one.

Reply:
As in a comment above, genuinely I would not want both to exist either, given the fact that:
* We need to allow BP users to define custom-made BP callbacks, thus the delegates they are bound to must be of `Dynamic` type.
* At the same time, regular delegates, due to being faster than dynamic, so I'd really want to make them as default and only expose ones with specific BP needs as Dynamic.
I haven't found out a better sustainable solution for this issue tho.

Reply:
As in a comment above, genuinely I would not want both to exist either, given the fact that:
* We need to allow BP users to define custom-made BP callbacks, thus the delegates they are bound to must be of `Dynamic` type.
* At the same time, regular delegates, due to being faster than dynamic, so I'd really want to make them as default and only expose ones with specific BP needs as Dynamic.
I haven't found out a better sustainable solution for this issue tho.

Reply:
This is a micro-optimization that obscures the API. I would just put it back to being a single dynamic delegate. If these callbacks ever show up on a profiler we could consider alternate options.

Reply:
I was kinda thinking about a scenario when we have like hundreds of robots, each with multiple publishers/messages, which might put more loads to the pixel streaming signal server + ros1bridge, etc. 
But yeah guess we could do some profiling then, for now, a single API definitely makes life easier.